go/base/management/commands/go_manage_routing_table.py

```diff
@@ -115,8 +115,8 @@ def print_routing_table(self, routing_table):
             return
 
         self.stdout.write("Routing table:\n")
-        for source, values in routing_table._routing_table.iteritems():
+        for source, values in sorted(routing_table._routing_table.iteritems()):
```

Comment:
Dict entries are unordered, and `.iteritems()` returns them in arbitrary order. Historically, we've had a bunch of tests that implicitly rely on ordering and have "accidentally" worked because the ordering is deterministic and the complicated state which determines it hasn't changed between test runs. When we run the tests on pypy, that's no longer true and the tests or this code fail because the routing table entries are displayed in a different order.

Sorting the entries before displaying them gives us a single correct ordering that we can assert on, and also makes it easier to find particular entries in the output.